specification/monitor/resource-manager/microsoft.insights/2017-11-01-preview/baseline_API.json

```diff
@@ -0,0 +1,347 @@
+                      "sensitivity": "Medium",
+                      "lowThresholds": [
+                        50.0,
+                        51.1
+                      ],
+                      "highThresholds": [
+                        70.3453,
+                        71.3453
+                      ]
+                    }
+                  ]                  
+                }
+              }
+            }
+          },
+          "x-ms-examples": {
+            "Get Metric for data": { "$ref": "./examples/GetBaseline.json" },
+            "Get Metric for metadata": { "$ref": "./examples/GetBaselineMetadata.json" }
+          }
+        }
+      }
+    },
+    "definitions": {
+      "LocalizableString": {
+        "required": [ "value" ],
+        "properties": {
+          "value": {
+            "type": "string",
+            "description": "the invariant value."
+          },
+          "localizedValue": {
+            "type": "string",
+            "description": "the locale specific value."
+          }
+        },
+        "description": "The localizable string class."
+      },
+      "BaselineProperties": {
+        "properties": {
+          "timespan": {
+            "type": "string",
+            "description": "The timespan for which the data was retrieved. Its value consists of two datatimes concatenated, separated by '/'.  This may be adjusted in the future and returned back from what was originally requested."
+          },
+          "interval": {
+            "type": "string",
+            "format": "duration",
+            "description": "The interval (window size) for which the metric data was returned in.  This may be adjusted in the future and returned back from what was originally requested.  This is not present if a metadata request was made."
+          },
+          "aggregation": {
+            "type": "string",
+            "description": "The aggregation type of the metric."
+          }
+        },
+        "description": "The baseline properties class."
+      },
+      "BaselineMetadataValue": {
+        "properties": {
+          "name": {
+            "$ref": "#/definitions/LocalizableString",
+            "description": "the name of the metadata."
+          },
+          "value": {
+            "type": "string",
+            "description": "the value of the metadata."
+          }
+        },
+        "description": "Represents a baseline metadata value."
+      },
+      "ErrorResponse": {
+        "description": "Describes the format of Error response.",
+        "type": "object",
+        "properties": {
+          "code": {
+            "description": "Error code",
+            "type": "string"
+          },
+          "message": {
+            "description": "Error message indicating why the operation failed.",
+            "type": "string"
+          }
+        }
+      },
+      "BaselineResponse": {
+        "type": "object",
+        "properties": {
+          "id": {
```

Comment:
id, name and type properties should be marked as readonly: true

Reply:
Added readonly:true